### PR TITLE
Support Arabic

### PR DESCRIPTION
**Pull request by dalwadani**
Add Arabic support:
```
>>> from num2words import num2words
>>> num2words(42, lang="ar")
'اثنين وأربعين'
>>> 
```

### CODE REVIEW

**Comment by erozqba** [comment]
Hello @dalwadani 
Thanks for taking the time to add Arabic support into mun2words. 
Could you please fix the encoding problem for Python 2.7 and add some test for Arabic like the one that exists for the other languages inside the tests directory?

**Comment by dalwadani** [comment]
Done!

**Comment by jerem** [comment]
Can this be merged/released now?

File: num2words/lang_AR.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    def to_ordinal_num(self, value):
+        self.verify_ordinal(value)
+        return "%s%s"%(value, self.to_ordinal(value)[-2:])
+
+
+    def to_year(self, val, longval=True):
+        if not (val//100)%10:
+            return self.to_cardinal(val)
+        return self.to_splitnum(val, hightxt="hundred", jointxt="and",
```
Comment (erozqba) [inline num2words/lang_AR.py]: I don't know any Arabic, but I don't think that you use "hundred" and "and", could you check this, please?

File: num2words/lang_AR.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                                longval=longval)
+
+    def to_currency(self, val, longval=True):
+        return self.to_splitnum(val, hightxt="dollar/s", lowtxt="cent/s",
+                                jointxt="and", longval=longval, cents = True)
+
```
Comment (erozqba) [inline num2words/lang_AR.py]: The same here, I see "dollar/s", "cent/s", "and".

**Comment by erozqba** [comment]
@dalwadani Sorry for taking too long in the review, could you please check the comments? Thanks!

**Comment by dalwadani** [comment]
Done!

**Comment by erozqba** [comment]
Thanks @dalwadani !

**Comment by jerem** [comment]
Thanks both of you!
@erozqba When can we expect a new release including this change?

**Comment by erozqba** [comment]
@jerem I will ask the main maintainer @hsoft about a new release as soon as possible.

**Comment by hsoft** [comment]
@dufresnedavid is the maintainer of the package now.

**Comment by dufresnedavid** [comment]
@hsoft Sorry, when I told you that I would maintain this repo, I was a bit out of context.

num2words is not relevant for Odoo development since Odoo version 9.0 (an equivalent of num2words is available directly in the source code of Odoo). Therefore, I don't see myself fit has maintainer of this project because I won't use it at all.

I would suggest @erozqba to take the flag.

**Comment by erozqba** [comment]
@hsoft I have made a new release v0.5.5. I try to publish it in https://pypi.python.org/pypi but I'm not allowed I get this error: Server response (403): You are not allowed to store 'num2words' package information
I have tested the release on the test server of PyPi, you could check it here: https://testpypi.python.org/pypi/num2words

Thanks in advance for your help and any suggestions or changes that must be made to the release.

**Comment by hsoft** [comment]
@erozqba I don't have permissions to publish PyPI packages under the SFL account any more so I can't help you much, but we had set up PyPI permissions with @dufresnedavid before I left.

**Comment by erozqba** [comment]
@jerem I have released the version v0.5.5

**Pull request merged**